<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P5XModTools</string>
  </property>
  <widget class="QProgressBar" name="downloadProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="launcherButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Launcher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fresh Install Cllient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Install Mod</string>
   </property>
  </widget>
  <widget class="QPushButton" name="autoDetectLocation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Autodetect Gamefile Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="gamefileLocationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gamefile Location</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="locationEditor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hotfixButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Install Hotfix</string>
   </property>
  </widget>
  <widget class="QComboBox" name="regionComboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KR</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="browserEditor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>621</width>
     <height>181</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="maximumBlockCount">
    <number>50</number>
   </property>
  </widget>
  <widget class="QPushButton" name="uninstallModButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Uninstall Mod</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>v1.0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>About This Program</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
